src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,190 @@
+package duke;
+
+// JavaFX Imports
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+public class Duke extends Application{
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+
+
+    }
+
+    /**
+     * Prints the greeting for the chatbot.
+     */
+    public void greeting() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        dialogContainer.getChildren().add(getDialogLabel("Hello from\n" + logo));
+    }
+
+    /**
+     * This method initiates the chatbot of the Duke program. It call upon various
+     * classes responsible for different functionalities of Duke.
+     */
+    @Override
+    public void start(Stage stage) {
+        Parser p = new Parser();
```

Comment:
Why don't you use the full name parser as the variable name? The same for others.